packages/utils/src/schema/toPathSchema.ts

```diff
@@ -15,7 +15,7 @@ import {
   RJSF_ADDITONAL_PROPERTIES_FLAG,
 } from '../constants';
 import { FormContextType, PathSchema, RJSFSchema, StrictRJSFSchema, ValidatorType } from '../types';
-import { getClosestMatchingOption } from './index';
+import getClosestMatchingOption from './getClosestMatchingOption';
```

Comment:
Whoopsie, this can cause an infinite import loop, so I went straight to the implementation. I think that IntelliJ tried to help me out by auto-importing this in a previous PR and we both missed it in the code review